### PR TITLE
feat(br): trigger br against diff files

### PR DESCRIPTION
**Pull request by purelind**
Trigger br&lighting integration tests against diff files. Only changes made in the "br/" directory will be automatically triggered.

### CODE REVIEW

File: prow-jobs/pingcap-tidb-latest-presubmits.yaml [hunk 1]
```diff
@@ -83,6 +83,7 @@ presubmits:
       decorate: false # need add this.
       always_run: false
       context: pull-br-integration-test
+      run_if_changed: "^br/.*$"
```
Comment (wuhuizuo) [inline prow-jobs/pingcap-tidb-latest-presubmits.yaml:86]:
```suggestion
      run_if_changed: "^br/.*$"
      skip_if_only_changed: "(\\.md|Dockerfile|OWNERS|OWNERS_ALIASES)$"      
```

**Review by wuhuizuo — APPROVED** [review]
/lgtm

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**